vmm/src/cpu.rs

```diff
@@ -884,7 +884,7 @@ impl CpuManager {
 
         #[cfg(target_arch = "x86_64")]
         let topology = self.config.topology.clone().map_or_else(
-            || Some((1, self.boot_vcpus(), 1)),
+            || Some((1, self.boot_vcpus() as u8, 1)),
```

Comment:
I know the `config.boot_vcpus` remains as `u8`. Let's make it safer by using `try_into().unwrap()`. 

Normally I'd suggest we use assertion since it indicates a programming error. I assume this will be changed soon in future patches, so `unwrap()` is fine with me.